### DISCUSSION TITLE
Easy way to deploy photon instead of lemmy-ui (ansible playbook ?) ?

### DISCUSSION THREAD

**Discussion by tifriis208** [Xyphyn/photon#142] [Ideas]
Hello,

I plan to create a lemmy instance running on rpi4 powered by solar energy (thanks for the arm64 support !!). I really prefer photon instead of lemmy-ui :)
Do yo know if there is an ansible playbook or other script to install photon instead of lemmy ui (with nginx configuration, ...) ?

It would be great if the ansible playbook could be play after [this one](https://github.com/LemmyNet/lemmy-ansible) to replace lemmy-ui by photon. I know a little bit ansible but I'm not confortable with docker/nginx for now

thanks !

**Comment by Xyphyn** [comment]
Photon has feature parity but it is definitely not recommended to use it as a replacement for lemmy-ui right now. I haven't implemented site creation, or password resets an email verification since those are handled by lemmy-ui right now.

I plan to make it a drop in replacement soon though.

**Reply by tifriis208** [reply]
Thanks for tour reply and advice ! 
Meanwhile, do you know if the le ils a way to install photon alongside lemmy-ui with ansible ?
Thanks again for this good work !

**Reply by Xyphyn** [reply]
Unfortunately I don't know a simple way as I have never looked at Ansible before. I'll check it out and see what I can do.

**Reply by tifriis208** [reply]
For information, some people are working on it : https://github.com/LemmyNet/lemmy-ansible/issues/180 :)
